service.tf

```diff
@@ -0,0 +1,24 @@
+resource "kubernetes_service" "bitbucket_runner" {
+  for_each = var.bitbucket_runners
+
+  metadata {
+    name      = "bitbucket-${each.key}"
+    namespace = var.k8s_namespace
+
+    labels = merge(local.k8s_default_labels, {
```

Comment:
`local.k8s_default_labels` is not defined within this module. We auto-inject it internally via our terragrunt framework, but that's not expected in a public module.